test/controllers/Task.js

```diff
@@ -71,4 +72,39 @@ describe('Task', () => {
     });
   });
 
+
+  describe('#assign', () => {
+    it('add assign', done => {
+      let path = '/tasks/' + taskFixtures.task1._id.toString();
+      let data = {assigned: ['2', '1']};
+      new Request()
+        .post(path)
+        .data(data)
+        .send()
+        .then(result => {
+          expect(result).to.equal('success');
+          done();
+        })
+        .catch(done);
+    });
+  });
+
+  describe('#assign', () => {
+    it('add assign', done => {
+      let path = '/tasks/' + taskFixtures.task1._id.toString();
+      let data = {assigned: ['3', '1']};
+      new Request()
+        .post(path)
+        .data(data)
+        .send()
+        .then(result => {
+          expect(result).to.equal('success');
+          done();
+        })
+        .catch(done);
+    });
+  });
+
+
+
```

Comment:
The second assign test case isn't neccesary. You can test the unique profileId insertion in first assign test case, just duplicate the profileIds, and see the result.